* chown & execle are used within uClibc in libc/stdlib/unix_grantpt.c

diff -Nur uClibc.orig/libc/sysdeps/linux/common/chown.c uClibc/libc/sysdeps/linux/common/chown.c
--- uClibc.orig/libc/sysdeps/linux/common/chown.c	Fri Mar 17 10:11:20 2006
+++ uClibc/libc/sysdeps/linux/common/chown.c	Sat Mar 18 10:25:11 2006
@@ -10,6 +10,8 @@
 #include "syscalls.h"
 #include <unistd.h>
 
+libc_hidden_proto(chown)
+
 #define __NR___syscall_chown __NR_chown
 static inline _syscall3(int, __syscall_chown, const char *, path,
 		__kernel_uid_t, owner, __kernel_gid_t, group);
@@ -23,3 +25,4 @@
 	}
 	return (__syscall_chown(path, owner, group));
 }
+libc_hidden_def(chown)
diff -Nur uClibc.orig/libc/unistd/exec.c uClibc/libc/unistd/exec.c
--- uClibc.orig/libc/unistd/exec.c	Fri Mar 17 10:11:02 2006
+++ uClibc/libc/unistd/exec.c	Sat Mar 18 10:23:45 2006
@@ -40,6 +40,7 @@
 #include <sys/mman.h>
 
 libc_hidden_proto(execl)
+libc_hidden_proto(execle)
 libc_hidden_proto(execvp)
 
 libc_hidden_proto(memcpy)
@@ -179,6 +180,7 @@
 
 	return n;
 }
+libc_hidden_def(execle)
 
 #endif
 /**********************************************************************/
